interfaces/entries-api.idl

```diff
@@ -44,25 +45,19 @@ dictionary FileSystemFlags {
     boolean exclusive = false;
 };
 
-callback interface FileSystemEntryCallback {
-    void handleEvent(FileSystemEntry entry);
-};
+callback FileSystemEntryCallback = void (FileSystemEntry entry);
 
 interface FileSystemDirectoryReader {
     void readEntries(FileSystemEntriesCallback successCallback,
                      optional ErrorCallback errorCallback);
 };
-callback interface FileSystemEntriesCallback {
-    void handleEvent(sequence<FileSystemEntry> entries);
-};
+callback FileSystemEntriesCallback = void (sequence<FileSystemEntry> entries);
 
 interface FileSystemFileEntry : FileSystemEntry {
     void file(FileCallback successCallback,
               optional ErrorCallback errorCallback);
 };
-callback interface FileCallback {
-    void handleEvent(File file);
```

Comment:
Yeah, this old pattern is just because of an oddity in Blink/WebKit bindings generations and should never have made it into any spec, glad it's gone!

(The resulting generated bindings *didn't* allow you to use `{ handleEvent: function(file) { ... } }` as the callback, so it was just a lie.)